README.md

```diff
@@ -1512,6 +1512,52 @@ would enable the use of a `@fold` on the `adjacent_animal` vertex field of `Foo`
 }
 ```
 
+## Schema Generation
+To generate a GraphQL schema, we first map a database schema into an intermediate representation:
+the `SchemaGraph`.
+
+#### SchemaGraph
+* `SchemaGraph`
+    * Represents a database's schema through `SchemaElement` objects.
+        * `SchemaElement`
+            * Can be one of three kinds:
+                * `EdgeType`
+                * `VertexType`
+                * `NonGraphElement`.
```

Comment:
What's a NonGraphElement? From what you've said, it's useful for representing inheritance. Could you add some explanation to this element?